src/org/whstsa/library/commands/functional/PopulateBooksCommand.java

```diff
@@ -0,0 +1,41 @@
+package org.whstsa.library.commands.functional;
+
+import org.json.JSONObject;
+import org.whstsa.library.api.library.ILibrary;
+import org.whstsa.library.commands.api.ICommand;
+import org.whstsa.library.commands.api.ICommandSender;
+import org.whstsa.library.db.ObjectDelegate;
+import org.whstsa.library.util.DayGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Created by Andre on 12/07/17.
+ */
+public class PopulateBooksCommand implements ICommand {
+    @Override
+    public JSONObject handle(String[] args, ICommandSender commandSender) {
+        int totalBooksAdded = Integer.parseInt(args[0]);
+        ILibrary libraryBeingPopulated = ObjectDelegate.getLibrary(UUID.fromString(args[1]));
```

Comment:
It doesn't look like you're ever using `libraryBeingPopulated`, is there a reason for it being declared?